### DISCUSSION TITLE
Rotating the imported image

### DISCUSSION THREAD

**Discussion by levankankadze** [xournalpp/xournalpp#3220] [Q&A] [answered]
Hello, 
Thank you for developing such an amazing soft.

I have question, is there any way to rotate imported image file in the document?
I am using it on windows and it is a last version.
Thank you

**Comment by rolandlo** [comment, marked as answer]
There is currently no way to rotate an imported image. In Xournal++ 1.1.0+dev it is possible to get rotated images via the LaTeX tool though. You just need to include the `graphics` and `rotating` packages in the LaTeX preamble and then type something like in the example below:

![grafik](https://user-images.githubusercontent.com/40485433/125161614-9f080380-e183-11eb-9253-ea283e0dc486.png)

**Reply by levankankadze** [reply]
Thank you! 
It is pity, hope in future we will have this feature.

**Reply by rolandlo** [reply]
It's already on the issue tracker https://github.com/xournalpp/xournalpp/issues/918
